types/copy-webpack-plugin/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for copy-webpack-plugin v4.0.0
+// Type definitions for copy-webpack-plugin v4.4.1
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
It seems to be disabled many rules in tslint.json.
I enable `dt-header` rule only.
https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md